pkg/plugins/golang/v3/scaffolds/webhook.go

```diff
@@ -93,6 +95,9 @@ func (s *webhookScaffolder) Scaffold() error {
 		&webhook.Kustomization{Force: s.force},
 		&webhook.KustomizeConfig{},
 		&webhook.Service{},
+		&certmanager.Certificate{},
+		&certmanager.Kustomization{},
+		&certmanager.KustomizeConfig{},
```

Comment:
we should not scaffold the certmanager by default. 
It is valid only for webhooks and then, should only be scaffolded when the project has webhooks.

Reply:
We should probably split this change to a different PR.

Reply:
that is the intention. It is wip :-)